src/systems/systemDialog.component.tsx

```diff
@@ -134,7 +134,7 @@ const SystemDialog = React.memo((props: SystemDialogProps) => {
           if (
             status === 409 &&
             errorMessage.includes(
-              'A System with the same name already exists within the same parent System'
+              'A system with the same name already exists within the parent system'
             )
           ) {
             setError('name', {
```

Comment:
Should the message itself not be updated too? I think most places we use the lower case now anyway e.g. `No system selected`, `Root system`.

Btw I found the issue https://github.com/ral-facilities/inventory-management-system/issues/1486, I have put it into up next and changed it to v3.